src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public abstract class Machine {
+
```

Comment:
empty line

Reply:
I see the notification "Rommelua requested changes". I did it. Also, I have seen that aaananas approved these changes [9 hours ago]. What I had to do else?

Reply:
My comment is about the empty line AFTER the class declaration - we do not need it.